mitodl/open-discussions
# Scope fixed-width form styles to auth pages

#### What are the relevant tickets?
N/A

#### What's this PR do?
Fixes a styling regression introduced in #824 

#### How should this be manually tested?
Go to the create a new post page and verify the form is the wider width from before the regression.

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
